.jenkinsci/selected-branches-coverage.groovy

```diff
@@ -1,13 +1,17 @@
 #!/usr/bin/env groovy
 
-def selectedBranchesCoverage(branches, PRCoverage=true) {
+def selectedBranchesCoverage() {
 	// trigger coverage if branch is either develop or master, or it is a PR
-	if (PRCoverage) {
-		return env.GIT_LOCAL_BRANCH in branches || env.CHANGE_ID != null
+	def pCommit = load ".jenkinsci/previous-commit.groovy"
+	def prevCommit = pCommit.previousCommitOrCurrent()
```

Comment:
`PREVIOUS_COMMIT` environment variable defined in Jenkinsfile can be used intstead.